### ISSUE TITLE
Document how to display a local file

### ISSUE THREAD

**Issue by kubilayeksioglu** [breach/pythrust#4]
Documentation displays how to open up http://breach.cc but there's no option to display a local file.

Currently this is what I'm using:

```
ROOT_FOLDER = os.path.abspath(os.path.dirname(__file__))
window = api.window({'root_url': 'file://%s/html/index.html' % ROOT_FOLDER})
```

but there might be a better option which I do not know.

**Comment by spolu** [comment]
This is correct. Feel free to open a pull request with documentation (even a small subset) if you want. You can look at the https://github.com/breach/thrust project for the documentation structure!
